### DISCUSSION TITLE
4.0.1 Release

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#3031] [General]
As a heads up, I am planning on releasing a 4.0.1 release this coming week. It will be a stabilization release which includes all of the recent changes/fixes.

**Comment by leigh-pointer** [comment]
Heads up, I will be testing :)

**Comment by vnetonline** [comment]
I will also test, Heads up :)

**Comment by sbwalker** [comment]
4.0.1 released: https://github.com/oqtane/oqtane.framework/releases/tag/v4.0.1
